py3-xmlsec.yaml

```diff
@@ -0,0 +1,61 @@
+package:
+  name: py3-xmlsec
+  version: 1.3.13
+  epoch: 0
+  description: About Python bindings for the XML Security Library
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - python3
+      - xmlsec
+      - xmlsec-openssl
+      - openssl
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python3
+      - python3-dev
+      - openssl-dev
+      - libffi-dev
+      - libretls
+      - libretls-dev
+      - libxslt-dev
+      - libxml2-dev
+      - xmlsec
+      - xmlsec-dev
+      - xmlsec-openssl
+      - py3-setuptools
+      - py3-gpep517
+      - py3-hatchling
+      - py3-pip
+      - git
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: b483b644b6033e539f303509f5ee85bebf178a73
+      repository: https://github.com/xmlsec/python-xmlsec
+      tag: ${{package.version}}
+
+  - uses: patch
+    with:
+      # Patch for open issue - https://github.com/xmlsec/python-xmlsec/issues/252
+      # https://github.com/xmlsec/python-xmlsec/pull/258
+      patches: pull_258.patch
+
+  - name: Python Install
+    runs: pip3 install . --root=${{targets.destdir}}
```

Comment:
Why not gpep517 here?  Using `pip` can be dangerous as it will vendor modules.

Reply:
Done. Switching to gpep517 also found a missing runtime dependency.